README.md

```diff
@@ -105,3 +105,35 @@ When you choose a name other than "editorconfig.json", enter the path of the edi
 ***
 
 After the above work, restart Firefox.
+
+***
+
+## Troubleshooting
+
+If something goes wrong, check the browser console (Ctrl + Shift + J).
+
+```
+Error: Attempt to postMessage on disconnected port
+```
+
+* Windows: Is the registory saved correctly?
+* Linux / Mac: Is "withexeditorhost.json" saved in the right location?
+* When you start Firefox, is Node.js process executed too?
```

Comment:
> is Node.js process executed too?

"Is a Node.js process executed too?" or "Is the Node.js process executed too?" (You need an article here in English)

In this situation, I'd recommend "a" since the user may have other Node.js applications.